includes/services/core/oauth1.php

```diff
@@ -89,46 +97,52 @@ function request_token() {
 							$this->callback_url
 						)
 					),
-				'scope' => $scope,
+				'scope'          => $scope,
 			),
 			$this->request_token_url
 		);
 
 		// Set up OAuth request
 		$req = $this->prepare_request( null, $this->request_token_method, $request_token_url, false );
 
-		$query = '';
+		$query  = '';
 		$parsed = parse_url( (string) $req );
-		if ( !empty( $parsed['query'] ) && 'POST' == strtoupper( $this->request_token_method ) ) {
+		if ( ! empty( $parsed['query'] ) && 'POST' === strtoupper( $this->request_token_method ) ) {
 			$request_token_url = str_replace( '?' . $parsed['query'], '', (string) $req );
-			$query = $parsed['query'];
+			$query             = $parsed['query'];
 		} else {
 			$request_token_url = (string) $req;
 		}
 
 		// Go and get a request token
 		switch ( strtoupper( $this->request_token_method ) ) {
-		case 'GET':
-			Keyring_Util::debug( "OAuth1 GET Request Token URL: $request_token_url" );
-			$res = wp_remote_get( $request_token_url );
-			break;
-
-		case 'POST':
-			Keyring_Util::debug( "OAuth1 POST Request Token URL: $request_token_url" );
-			Keyring_Util::debug( $query );
-			$res = wp_remote_post( $request_token_url, array( 'body' => $query, 'sslverify' => false ) );
-			break;
-
-		default:
-			Keyring::error( __( 'Unsupported method specified for request_token.', 'keyring' ) );
-			exit;
+			case 'GET':
+				Keyring_Util::debug( "OAuth1 GET Request Token URL: $request_token_url" );
+				$res = wp_remote_get( $request_token_url );
+				break;
+
+			case 'POST':
+				Keyring_Util::debug( "OAuth1 POST Request Token URL: $request_token_url" );
+				Keyring_Util::debug( $query );
+				$res = wp_remote_post(
+					$request_token_url,
+					array(
+						'body'      => $query,
+						'sslverify' => false,
+					)
+				);
+				break;
+
+			default:
+				Keyring::error( __( 'Unsupported method specified for request_token.', 'keyring' ) );
+				exit;
 		}
 
 		Keyring_Util::debug( 'OAuth1 Response' );
 		Keyring_Util::debug( $res );
 
 		// Accept all 2xx response codes
-		if ( '2' == substr( wp_remote_retrieve_response_code( $res ), 0, 1 ) ) {
+		if ( '2' === substr( wp_remote_retrieve_response_code( $res ), 0, 1 ) ) {
```

Comment:
This changes the logic too, but `substr` will always return a string, so it should be fine.